Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+RUN apt-get update \
+    # Install gcc for x86 and ARM
+    && apt-get install -y build-essential gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git \
+    curl gawk texinfo expat libexpat1-dev bison flex libz-dev python \
+    # Prepare to download and compile RISCV toolchain
+    && export HOME_DIR=/home/root \
```

Comment:
There's a "real" way to do this in docker (`ENV` maybe?). I forget exactly.

Reply:
I forgot to change it. Will use ENV.

Reply:
solved